content/rekor-v2-ga.md

```diff
@@ -0,0 +1,166 @@
+
+If you're building a client, you MUST NOT hardcode this URL. We will periodically rotate the log instance, and while
+we'll publicly communicate when we spin up a new log, we will also freeze the previous log instance shortly afterwards.
+
+### How do I upload entries to the public instance?
+
+The easiest way is to use an updated Sigstore client or SDK.
+
+If you want to experiment with the API, we've provided an
+[example](https://github.com/sigstore/rekor-tiles/blob/main/CLIENTS.md#rekor-v2-the-bash-way). Note
+that Rekor v2 exposes only a single endpoint, `/api/v2/log/entries`.
+
+### How do I read entries from the public instance?
+
+If you're monitoring the log, the easiest way is to use [rekor-monitor](https://github.com/sigstore/rekor-monitor).
+
+Read requests are served by a standardized [read API](https://github.com/C2SP/C2SP/blob/main/tlog-tiles.md#merkle-tree)
+that serves tiles and entries. Note that these APIs are significantly different than Rekor v1, namely
+there are no Get-By-Log-Index or Get-By-Leaf-Hash APIs. The Sigstore clients handle the logic to fetch the set of tiles
+needed to compute an inclusion proof given a log index.
+
+Unless you're monitoring the log for entries, we discourage using the read API for computing inclusion proofs. The log
+will return inclusion proofs on upload and these should be persisted alongside artifacts and their signatures.
+
+Note that the Rekor v2 server does expose a read API, but this is not currently implemented and will only be
+implemented for specific storage backends.
+
+### What are the benefits of Rekor v2 over v1 as a user?
+
+Rekor v2 is more easily scalable so the public instance will support a higher QPS.
+
+Rekor v2 also will provide stronger security guarantees that the log remains append-only
+by integrating [witnessing](https://blog.transparency.dev/can-i-get-a-witness-network) directly into Rekor.
+This will be implemented soon - Reach out if you have more questions on this topic.
+
+### What are the benefits of Rekor v2 over v1 as a private operator?
+
+You will see both a reduction in storage costs from the [tile-based backend](https://transparency.dev/articles/tile-based-logs/)
+and infrastructure cost as you can turn down Trillian log server and log signer instances. You will also see a reduction
+in egress costs for any read requests since they're fully cacheable and can be served by a CDN.
+
+### What's been deprecated between v1 and v2?
+
+The log no longer returns signed timestamps with proofs. Sigstore clients will fetch
+a signed timestamp from a dedicated service. We've deployed a public instance timestamp authority,
+and users can configure their signing workflows to use other public or private timestamp authorities.
+
+The search index has been removed, since this was a best-effort service. This will be
+implemented as a dedicated service later on depending on the community's needs.
```

Comment:
We have it scheduled for 2.1 in our plans, and I think it's been voiced that this is an important need

Reply:
I don't want to commit to anything at the moment, though I'll mention we'll be reimplementing this as a verifiable service, and that the value of this service is to simplify monitoring, and we can engage with the community to determine what are the most important indexes beyond identity.